taskcluster/docker/base/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM ubuntu:18.04
+#rebuild13
```

Comment:
What is this `rebuild13` referring to, btw?

Reply:
CoT is unhappy when I rewrite commits such that the commit the docker image was built on is no longer a commit in the pull-request, so I've been forcing it to rebuild. This will go away when this lands.